app/src/components/Shared/ToolTip.jsx

```diff
@@ -57,34 +57,42 @@ class ToolTip extends Component {
 
   showToolTip() {
     const bounds = this.refs.info.getBoundingClientRect();
-
     const left = bounds.left; // Horizontal position relative to the screen
-    const height = bounds.height;
-    const width = bounds.width;
-
-    // Position relative to the abbr element
-    const offsetTop = this.refs.info.offsetTop;
-
-    // If true, the arrow is on the right of the tooltip
+    const height = bounds.height; // Height icon
+    const width = bounds.width; // Width icon
+    const offset = 10; // Distance between tooltip and icon
+
+    const top = bounds.top + window.scrollY + height + offset; // Calc position top normal tooltip
+    const topRight = bounds.top + window.scrollY - 40 + (height / 2); // Calc position top tooltip with arrow at right
```

Comment:
Can you explain where those 40, 100 and 200 values come from? Is it possible to have them in a variable?